doc/user/content/overview/_index.md

```diff
@@ -10,185 +10,84 @@ menu:
-complex tasks.
-
-By modeling a SQL query as a dataflow, you can take in a CDC stream, apply a set
-of transformations to it, and then observe the results as final operator's
-output set.
-
-### Putting it all together
-
-To put all of this together, Materialize lets you:
-
-- Describe queries you want to know the answer to as **materialized views**,
-  which is a concept implemented in **SQL**.
-- Use CDC **streams** to feed our dataflow **engine** to create and maintain the
-  answers to those queries.
-
-Ultimately, this lets you refresh answers to your queries very quickly because
-Materialize continually updates their answers as new data streams in.
-
-## How does Materialize work?
-
-While we've covered the high-level details above, this section gives some of the
-more specific details about Materialize itself.
-
-Materialize ingests streams of data from external systems like Kafka and
-PostgreSQL that you declare as "sources". These sources monitor changes in the
-upstream data and automatically pull new records into Materialize. You can
-interact with this data in Materialize as if it were in a SQL table.
-
-Using these sources, you can define "views". Views represent those queries to
-which you continually want an up-to-date answer. Your view definitions are
-transformed by our query engine to create [Differential dataflows](https://github.com/frankmcsherry/differential-dataflow).
-
-A dataflow is a set of connected operators that each both consume and produce
-streams of data, creating a network of computation that can respond to streams
-of input data from sources. Differential dataflows are special, though, in that
-they can easily perform incremental updates at each step of the dataflow.
-
-As data streams in from your sources, your dataflows determine which data is
-relevant and update their result sets only if they need to, for example, when
-there are new rows or the values used in dataflow computations have changed.
-
-When you query one of your views, Materialize can simply return the dataflow's
-result set, which should always be faster than computing the answer from
-scratch.
-
-## Materialize vs. other methodologies
-
-### Batch processing
-
-Batch processing relies on performing large, expensive computations
-infrequently. This means that you can never achieve real-time results, and it
-often does not build upon its prior computations.
-
-In contrast, Materialize continually updates queries as data comes in, which
-means it performs small, efficient computations constantly. This enables
-real-time results and builds upon all of your prior computations.
-
-### Materialized views in relational databases
-
-To maintain materialized views, most RDBMSes occasionally re-run the view's
-underlying query. This results in a potential impact to the database's
-performance while updating the view, as well as data that is only infrequently
-up-to-date.
+<hr>
```

Comment:
This is missing a wrap-up with a CTA, but I'll leave that and bridging between sections to your best judgement, @jseldess! If we think it's valuable, I can add a very brief section for use cases later, using the following structure:

```
## Use cases

### Real-Time analytics

### Automation and alerting

### Segmentation and personalization
```